### PR TITLE
Deduplicate documentation

### PR DESCRIPTION
**Pull request by backus**

### CODE REVIEW

**Comment by backus** [comment]
@andyw8 could you review these changes? Details in #173

File: Rakefile [hunk 1]
```diff
@@ -27,4 +30,19 @@ task :internal_investigation do
   abort('RuboCop failed!') unless result.zero?
 end
 
-task default: [:spec, :internal_investigation]
+desc 'Build config/default.yml'
+task :build_config do
+  sh('bin/build_config')
+end
+
+desc 'Confirm config matches is unchanged'
```
Comment (andyw8) [inline Rakefile]: "are unchanged"?

File: bin/build_config [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift(__dir__ + '/../lib')
```
Comment (andyw8) [inline bin/build_config]: Should this use `File.join` for Windows compatibility?

**Comment by andyw8** [comment]
(Incomplete review, will look again later today).

File: bin/build_config (hunk 2 above)
Comment (backus) [inline bin/build_config]: Good idea

File: Rakefile [hunk 3]
```diff
@@ -27,4 +30,19 @@ task :internal_investigation do
...
+task confirm_config: :build_config do
+  _, stdout, _, process =
+    Open3.popen3('git diff --exit-code config/default.yml')
+
+  if ENV.key?('CI') && !process.value.success?
```
Comment (andyw8) [inline Rakefile]: Any reason to limit this to CI? Might be helpful to have it available locally?

File: config/default.yml [hunk 4]
```diff
@@ -46,82 +45,82 @@ RSpec/ExampleWording:
...
-  Description: 'Checks the file and folder naming of the spec file.'
+  Description: Checks that spec file paths are consistent with the test subject
   Enabled: true
   CustomTransform:
     RuboCop: rubocop
     RSpec: rspec
 
 RSpec/VerifiedDoubles:
-  Description: 'Prefer using verifying doubles over normal doubles.'
+  Description: Prefer using verifying doubles over normal doubles.
```
Comment (andyw8) [inline config/default.yml]: Looks like some of these end with periods, others don't. Worth unifiying?

File: lib/rubocop/cop/rspec/let_setup.rb [hunk 5]
```diff
@@ -3,7 +3,7 @@
 module RuboCop
   module Cop
     module RSpec
-      # Detect unreferenced `let!` calls being used for test setup
+      # Checks for `let!` being used for test setup.
```
Comment (andyw8) [inline lib/rubocop/cop/rspec/let_setup.rb]: I think the previous message was better. `let!` is ok for test setup as long as the name is referenced.

File: config/default.yml (hunk 4 above)
Comment (backus) [inline config/default.yml]: Sure I was actually going to and got bogged down by other changes in this PR which I ended up abandoning (tl;dr splitting up config like rubocop does into `config/enabled.yml` and `config/default.yml` is impossible with the current monkey patching we do)

File: Rakefile (hunk 3 above)
Comment (backus) [inline Rakefile]: Hm well I just didn't want it to fail the rake task unless we were on CI. It still dumps the new config locally so you would see it while staging your code

File: Rakefile (hunk 1 above)
Comment (backus) [inline Rakefile]: Good point I just ended up rewording it

File: Rakefile (hunk 3 above)
Comment (backus) [inline Rakefile]: Alright I just made it fail everywhere if it is out of sync

File: lib/rubocop/cop/rspec/let_setup.rb (hunk 5 above)
Comment (backus) [inline lib/rubocop/cop/rspec/let_setup.rb]: Updated. FYI the main thing I was going for was trying to standardize messages using "Check" or "checks" when possible. You are right though

File: config/default.yml (hunk 4 above)
Comment (backus) [inline config/default.yml]: Unified now

**Comment by backus** [comment]
@andyw8 ready for another pass

**Comment by backus** [comment]
@andyw8 any further comments?

**Comment by andyw8** [comment]
All looks good to me.

**Pull request merged**